pkg/oauth/tokenrequest/request_token.go

```diff
@@ -87,8 +87,8 @@ func RequestTokenWithChallengeHandlers(clientCfg *restclient.Config, challengeHa
 // authzURLHandler is used to forward the user to the OAuth server's parametrized authorization URL to
 // retrieve the authorization code.
 // It starts a localhost server on port `callbackPort` (random port if unspecified) to exchange the authorization code for an access token.
-func RequestTokenWithLocalCallback(clientCfg *restclient.Config, authzURLHandler AuthorizationURLHandlerFunc, callbackPort int) (string, error) {
-	o, err := NewRequestTokenOptions(clientCfg, false).WithLocalCallback(authzURLHandler, callbackPort)
+func RequestTokenWithLocalCallback(clientCfg *restclient.Config, authzURLHandler AuthorizationURLHandlerFunc, callbackPort int, clientID string) (string, error) {
```

Comment:
We could move the `clientID` argument right after `clientCfg` for the sake of consistency with `RequestTokenWithChallengeHandlers`.